### ISSUE TITLE
Best Practices for Wrangler Example

### ISSUE THREAD

**Issue by Shaun-Regenbaum** [solidjs/solid-service-api#3]
Your cloudflare account is still in the wrangler, which it probably should not be :P

We should probably put wrangler.toml in .gitignore and instead make a example.wrangler.toml file with the proper structure for people to reference.

I can do this later if you remind me 😉

**Comment by davedbase** [comment]
Yeah, that doesn't bother me much. It would be a different story if it was the token itself. Haha. Plus we're migrating to a new account soon at which point that will be a defunct value.
